### PR TITLE
Support for Stable LM 2

### PR DESCRIPTION
**Pull request by dakotamahan-stability**
- add config.qkv_bias from Stable LM 2 to modeling code

### CODE REVIEW

File: vllm/model_executor/models/stablelm.py [hunk 1]
```diff
@@ -98,7 +98,11 @@ def __init__(self,
         self.scaling = self.head_dim**-0.5
         self.q_size = self.num_heads * self.head_dim
         self.kv_size = self.num_key_value_heads * self.head_dim
-
+        try:
+            self.qkv_bias = config.use_qkv_bias
+        except AttributeError:
+            # To support stablelm-3b-4e1t, which has a different config.
+            self.qkv_bias = False
```
Comment (zhuohan123) [inline vllm/model_executor/models/stablelm.py:105]:
```suggestion
        self.qkv_bias = getattr(config, "use_qkv_bias", False)
```

Comment (zhuohan123) [reply vllm/model_executor/models/stablelm.py:105]: This is probably simpler?

**Review by zhuohan123 — COMMENTED** [review]
In general LGTM! Can you provide an example huggingface model for testing this?

**Review by zhuohan123 — APPROVED** [review]

**Comment by dakotamahan-stability** [comment]
Oh sure, here's a stable lm 2 1.6b model: https://huggingface.co/stabilityai/stablelm-2-zephyr-1_6b

**Pull request merged**